samples/test/transcoder.test.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * Copyright 2020, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
```

Comment:
To fix the linter errors you can add the following import
```
import {describe, it} from 'mocha';
```

See https://github.com/googleapis/nodejs-video-transcoder/blob/master/test/gapic_transcoder_service_v1beta1.ts#L23

Reply:
Oops, since this file is JS and not TS, it should actually be:
```
const {describe, it} = require('mocha');
```

Reply:
Thanks! That worked.